.readthedocs.yaml

```
@@ -0,0 +1,22 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: '3.8'
```

```suggestion
  os: ubuntu-22.04
  tools:
    python: "3.12"
```
Let's use the latest